### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by WalrusInAnus** [RevoLand/Steam-Library-Manager#16]
I downloaded the latest version, but it's crashing in startup.
I have .NET 4.7.1 installed.
EventLog says:

```
Faulting application name: Steam.Library.Manager.exe, version: 1.4.4.3, time stamp: 0x5a4139c1
Faulting module name: clr.dll, version: 4.7.2558.0, time stamp: 0x59d414b7
Exception code: 0xc00000fd
Fault offset: 0x0000000000003107
Faulting process id: 0x4b4
Faulting application start time: 0x01d37e3d0d7b5112
Faulting application path: D:\Gaming stuff\Steam.Library.Manager.exe
Faulting module path: C:\Windows\Microsoft.NET\Framework64\v4.0.30319\clr.dll
Report Id: 4be3c44b-ea30-11e7-a14f-902b34ddceaf
```

**Comment by RevoLand** [comment]
Hello,

Sorry about the crash you had, i have fixed it with e12442d, can you please re-download [v1.4.4.3](https://github.com/RevoLand/Steam-Library-Manager/releases/tag/v1.4.4.3) and test?

**Comment by WalrusInAnus** [comment]
All good!
